code/modules/mob/living/basic/jungle/venus_human_trap.dm

```diff
@@ -163,74 +175,72 @@
 	faction = list(FACTION_HOSTILE,FACTION_VINES,FACTION_PLANTS)
 	initial_language_holder = /datum/language_holder/venus
 	unique_name = TRUE
-	/// A list of all the plant's vines
-	var/list/vines = list()
-	/// The maximum amount of vines a plant can have at one time
-	var/max_vines = 4
-	/// How far away a plant can attach a vine to something
-	var/vine_grab_distance = 5
-	/// Whether or not this plant is ghost possessable
-	var/playable_plant = TRUE
+	speed = 1.2
+	melee_attack_cooldown = 1.2 SECONDS
+	ai_controller = /datum/ai_controller/basic_controller/human_trap
 
-/mob/living/simple_animal/hostile/venus_human_trap/Life(seconds_per_tick = SSMOBS_DT, times_fired)
+/mob/living/basic/venus_human_trap/Initialize(mapload)
 	. = ..()
-	pull_vines()
+	AddElement(/datum/element/lifesteal, 5)
+	var/datum/action/cooldown/vine_tangle/tangle = new(src)
+	tangle.Grant(src)
+	ai_controller.set_blackboard_key(BB_TARGETTED_ACTION, tangle)
 
-/mob/living/simple_animal/hostile/venus_human_trap/Moved(atom/old_loc, movement_dir, forced, list/old_locs, momentum_change = TRUE)
-	. = ..()
-	pixel_x = base_pixel_x + (dir & (NORTH|WEST) ? 2 : -2)
+/mob/living/basic/venus_human_trap/RangedAttack(atom/victim)
+	if(!combat_mode)
+		return
+	var/datum/action/cooldown/mob_cooldown/tangle_ability = ai_controller.blackboard[BB_TARGETTED_ACTION]
+	if(!istype(tangle_ability))
+		return
+	tangle_ability.Trigger(target = victim)
 
-/mob/living/simple_animal/hostile/venus_human_trap/AttackingTarget()
+/mob/living/basic/venus_human_trap/Life(seconds_per_tick = SSMOBS_DT, times_fired)
 	. = ..()
-	if(isliving(target))
-		var/mob/living/L = target
-		if(L.stat != DEAD)
-			adjustHealth(-maxHealth * 0.1)
-
-/mob/living/simple_animal/hostile/venus_human_trap/OpenFire(atom/the_target)
-	for(var/datum/beam/B in vines)
-		if(B.target == the_target)
-			pull_vines()
-			ranged_cooldown = world.time + (ranged_cooldown_time * 0.5)
-			return
-	if(get_dist(src,the_target) > vine_grab_distance || vines.len >= max_vines)
+	if(!.)
+		return FALSE
+
+	var/vines_in_range = locate(/obj/structure/spacevine) in range(2, src)
+	if(!vines_in_range)
+		balloon_alert(src, "do not leave vines!")
+	apply_damage(20 * (vines_in_range ? -0.5 : 1), BRUTE, BODY_ZONE_CHEST) //every life tick take 20 brute if not near vines or heal 10 if near vines, 5 times out of weeds = u ded
+
+/datum/action/cooldown/vine_tangle
+	name = "Tangle"
+	button_icon = 'icons/mob/spacevines.dmi'
+	button_icon_state = "Light1"
+	desc = "Trip and pull things."
```

Comment:
```suggestion
	desc = "Grabs a target with a sticky vine, allowing you to pull it alongside you."
```
If you're going to make it need manual reeling by physically moving yourself at least hint at it in the ability tooltip.